app/assets/stylesheets/custom.scss

```
@@ -12,6 +12,10 @@
 
   .question-answer-fields {
     margin-bottom: $line-height;
+
+    &.single-choice .form-error {
+      margin-top: inherit;
```

The `inherit` keyword can be hard to understand when talking about margins :thinking:.

```suggestion
      margin-bottom: 0;
      margin-top: 0;
```

I'm also suggesting the `margin-bottom` line because it seems to me that we're getting an extra margin between form fields when there's an error :thinking:.